internal/importers/base/importer.go

```diff
@@ -132,7 +132,11 @@ func (i *Importer) loadPackages(packages []ImportedPackage) ([]importedProject,
 	for _, pkg := range packages {
 		pr, err := i.SourceManager.DeduceProjectRoot(pkg.Name)
```

Comment:
@carolynvs When does `SourceManager.DeduceProjectRoot` return an error? I tried passing a package path which does not exist but that did not return an error.

Reply:
`not-a-valid-project` works. Let me know if you want me to use something else.